mappings/net/minecraft/data/client/model/Texture.mapping

```diff
@@ -1,20 +1,20 @@
 CLASS net/minecraft/class_4944 net/minecraft/data/client/model/Texture
 	FIELD field_22997 entries Ljava/util/Map;
 	FIELD field_22998 inherited Ljava/util/Set;
-	METHOD method_25860 getModelId (Lnet/minecraft/class_2248;)Lnet/minecraft/class_2960;
+	METHOD method_25860 getId (Lnet/minecraft/class_2248;)Lnet/minecraft/class_2960;
```

Comment:
Why this change? Specifying that it's the model id make it more clear

Reply:
this can actually be texture id as well